### PR TITLE
German Locale

### PR DESCRIPTION
**Pull request by MetalMatze**
Closed #1004 , rebasing gone wrong somehow.
Now this is actually rebased.

👍 from me!

### CODE REVIEW

**Comment by MarshallOfSound** [comment]
@MetalMatze This PR is failing tests due to a missing translation key

```js
AssertionError: expected { Object (audio-device-communications, audio-device-default, ...) } to have a property 'settings-option-auto-launch'
```

File: src/_locales/de.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  "settings-option-enable-system-borders": "Benutze Fensterrahmenthema des Systems",
+  "settings-option-enable-system-borders-details": "Erfordert einen Neustart",
+  "settings-option-enable-voice": "Aktiviere Sprachsteuerung",
+  "settings-option-enable-voice-details": "Sprachsteuerung für GPMDP, dies ist experimentell",
+
+  "settings-option-hotkey-explanation": "Die grundlegenden Medientasten werden dennoch funktionieren.  Diese Einstellungen kommen zu den Standardeinstellungen hinzu.",
+  "settings-option-hotkey-explanation-2": "Zum Zurücksetzen der Tastenkombination benutze die <i>\"Escape\"</i>-Taste.",
+
+  "settings-option-min-to-tray": "In den Infobereich minimieren",
+
```
Comment (berryelectronics) [inline src/_locales/de.json]: `"settings-option-auto-launch": "Automatisch starten wenn mein Computer hochfährt",`

fehlt noch, ist erst seit kurzer Zeit im master.

**Comment by Bonez76** [comment]
Hope you don´t mind some suggestions.

**> "lastfm-logout-button-text": "Von Last.FM abmelden",**
Maybe better "Bei Last.FM abmelden"

**> "lyrics-failed-message": "Entschuldige, aber wir konnten keinen Liedtext finden"**
A bit long. What about "Leider keinen Liedtext gefunden" or simply "Kein Liedtext gefunden"

**> "modal-confirmUninstall-title": "Hier ist das Problem...",**
When does this text appear? 

**> "modal-welcome-title": "Willkommen zur Version",**
When does this text appear? 

**> "settings-option-enable-system-borders": "Benutze Fensterrahmenthema des Systems",**
What about "Verwende Farbeinstellungen des Systems"?

**> "settings-option-enable-voice-details": "Sprachsteuerung für GPMDP, dies ist experimentell",**
"Sprachsteuerung für GPMDP (experimentell)" just like in songtext feature with "(beta)"

**> "settings-option-auto-launch": "Automatisch starten wenn mein Computer hochfährt",**
What about ""Automatisch beim Start von Windows laden"?

**> "tray-label-audio-device": "Audiogerät",**
"Wiedergabegerät"

**> "title-settings-mini":** "Mini Player",
missing a "-" > "Mini-Player"

I general: I would avoid "!" or "!!" at the end of an text. Not very good style :) Same with "..." at the end of a text.

Hope I did this correct...am new to Github.

**Comment by Bonez76** [comment]
Made a commit with my suggestions. [https://github.com/Bonez76/Google-Play-Music-Desktop-Player-UNOFFICIAL-/commit/26b820095334f1e43401d762e3f6d0b2035577c1](url)

**Comment by MarshallOfSound** [comment]
@MetalMatze The translations file has had two new string added to it.  Can you make a new PR with those two new translations please 👍 

```js
{
...
  "modal-confirmOpenPort-content": "",
...
  "settings-option-mini-use-scroll-volume": "",
...
}
```

**Comment by MetalMatze** [comment]
Oh, I didn't see it... Sorry. I'll try to update on the weekend.

**Pull request merged**